### PR TITLE
Sorts Choose Spouse (Mate) and Special Ability lists

### PR DESCRIPTION
**Pull request by bandildo**
Implements #475 
Also added scroll pane to award list.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java [hunk 1]
```diff
@@ -1814,6 +1820,10 @@ private void maybeShowPopup(MouseEvent e) {
                         menuItem.addActionListener(this);
 
                         setAwardMenu.add(menuItem);
+
+                        if (setAwardMenu.getItemCount() > 30) {
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java]: Should this be smaller? We've had requests by folks with small vertical resolutions elsewhere.

Comment (bandildo) [reply MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java]: I'll change it to the 20 items like the spouse one, assuming that value is ok.

Comment (sixlettervariables) [reply MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java]: I'll double check on my 13" MacBook today.

Comment (sixlettervariables) [reply MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java]: You still left the `if` test at 30.

**Comment by bandildo** [comment]
Sorry, brainfart here. Hopefully all is okay now.

**Pull request merged**